### PR TITLE
Translation french jenkins2 various screens

### PR DESCRIPTION
**Pull request by jpbriend**
Add missing french translations to Manage New Item, Jenkins screen, Global Tools screen.

### CODE REVIEW

File: core/src/main/resources/jenkins/model/Jenkins/configure.jelly [hunk 1]
```diff
@@ -62,7 +62,7 @@ THE SOFTWARE.
 
       <f:bottomButtonBar>
         <f:submit value="${%Save}" />
-        <f:apply />
+        <f:apply value="${%Apply}"/>
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/model/Jenkins/configure.jelly]: Why this change?

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy [hunk 2]
```diff
@@ -35,7 +35,7 @@ l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName) {
 
             f.bottomButtonBar {
                 f.submit(value:_("Save"))
-                f.apply()
+                f.apply(value:_("Apply"))
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy]: Why this change?

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Back\ to\ Dashboard=Back to Dashboard
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties]: Untranslated strings.

File: core/src/main/resources/jenkins/model/ProjectNamingStrategy/PatternProjectNamingStrategy/config_fr.properties [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# The MIT License
+#
+# Copyright (c) 2012, Dominik Bartholdi
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/model/ProjectNamingStrategy/PatternProjectNamingStrategy/config_fr.properties]: Not true.

File: core/src/main/resources/jenkins/model/GlobalProjectNamingStrategyConfiguration/config_fr.properties [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# The MIT License
+#
+# Copyright (c) 2012, Dominik Bartholdi
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/model/GlobalProjectNamingStrategyConfiguration/config_fr.properties]: Not true.

File: core/src/main/resources/hudson/model/View/newJob_fr.properties [hunk 6]
```diff
@@ -23,3 +23,10 @@
 NewJob=Nouveau {0}
 JobName=Nom du {0}
 CopyExisting=Copier un {0} existant
+ItemName.help=Champ obligatoire
+ItemName.label=Saisissez un nom
+ItemName.validation.required=Ce champ ne peut pas \u00eatre vide. Veuillez saisir un nom valide et appuyer sur OK.
+ItemType.validation.required=Veuillez choisir  un type d' Item
```
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/View/newJob_fr.properties]: Looks like the file type doesn't like single quotes?

**Comment by daniel-beck** [comment]
@batmat @aheritier Ping!

File: core/src/main/resources/hudson/model/View/newJob_fr.properties (hunk 6 above)
Comment (batmat) [inline core/src/main/resources/hudson/model/View/newJob_fr.properties]: "choisir  un" unintended double space?

File: core/src/main/resources/jenkins/model/Messages_fr.properties [hunk 7]
```diff
@@ -42,6 +42,12 @@ Hudson.ReadPermission.Description=\
   &mdash; retirez ce droit \u00e0 l''utilisateur anonymous, puis \
   ajoutez le pseudo-utilisateur "authenticated" et accordez-lui le droit en lecture.
 
+Mailer.Address.Not.Configured=Adresse pas encore configur\u00e9ea <nobody@nowhere>
```
Comment (batmat) [inline core/src/main/resources/jenkins/model/Messages_fr.properties]: Not sure, but that's why I hate encoded strings when it can be avoided. Shouldn't it be `configur\u00e9e` since `\u00e9` stands for é and then the final e? Hence the a is to be removed?

File: core/src/main/resources/jenkins/model/Messages_fr.properties [hunk 8]
```diff
@@ -42,6 +42,12 @@ Hudson.ReadPermission.Description=\
...
+Mailer.Localhost.Error=Veuillez configurer un nom d''h\u00f4te valide, au lieu de localhost
+
+PatternProjectNamingStrategy.DisplayName=Pattern
+PatternProjectNamingStrategy.NamePatternRequired=Le nom de pattern est obligatoire
+PatternProjectNamingStrategy.NamePatternInvalidSyntax=La syntaxe de l''expression r\u00e9guli\u00e8re est invalide.
 ParameterizedJobMixIn.build_now=Lancer un build
 BlockedBecauseOfBuildInProgress.shortDescription=Le build #{0} est d\u00e9j\u00e0 en cours {1}
 BlockedBecauseOfBuildInProgress.ETA=\ (fin pr\u00e9vue \u00e0: {0})
```
Comment (batmat) [inline core/src/main/resources/jenkins/model/Messages_fr.properties]: missing unbreakable space before column

File: core/src/main/resources/jenkins/model/ProjectNamingStrategy/PatternProjectNamingStrategy/config_fr.properties [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+namePattern = Pattern du nom
```
Comment (batmat) [inline core/src/main/resources/jenkins/model/ProjectNamingStrategy/PatternProjectNamingStrategy/config_fr.properties]: NIT: not the same pattern as above, with space around the `=`. On purpose?

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties (hunk 3 above)
Comment (batmat) [inline core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties]: @daniel-beck it's the english version, see next file?

Comment (daniel-beck) [inline core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties]: @batmat That makes sense :smile:

File: core/src/main/resources/jenkins/model/Jenkins/configure.jelly (hunk 1 above)
Comment (jpbriend) [inline core/src/main/resources/jenkins/model/Jenkins/configure.jelly]: In order to be able to translate the Apply button. If no value is provided, 'Apply'is not translated.

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy (hunk 2 above)
Comment (jpbriend) [inline core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy]: Same as above.

**Comment by jpbriend** [comment]
ping @batmat @daniel-beck  for review

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties (hunk 3 above)
Comment (daniel-beck) [inline core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties]: @batmat  But… why!?

**Comment by aheritier** [comment]
You have the copyright to fix in ...ojectNamingStrategyConfiguration/config_fr.properties @jpbriend 
Otherwise :bee:

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties (hunk 3 above)
Comment (batmat) [inline core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties]: Why wut? You mean this file isn't even necessary cause it's defined by default in the corresponding files? (Cannot check now on the phone).

**Comment by jpbriend** [comment]
@aheritier thank you for the review !

**Comment by oleg-nenashev** [comment]
Seems it's ready to go, right?

**Comment by aheritier** [comment]
yes for me it is good
